docusaurus.config.js

```diff
@@ -108,7 +108,8 @@ module.exports = {
     },
     image: 'img/favicon.png',
     algolia: {
-      apiKey: '038c83f2ea7d91ea50d9e914396ec1b5',
+      apiKey: '1a42b1ebe30454fa429a47c5ed59de94',
```

Comment:
Could we maybe set the apiKey as part of the deployment process using github secrets to avoid exposure of key in public repository? (e.g. step in deployment actions which replaces placeholder with secret)

I assume it's an free account with limits, so it's not that critical, but I would still prefer a clean solution.

Reply:
According to feedback it will be public anyways since hosted as static page.